description.md

```diff
@@ -30,13 +32,13 @@ Put differently, it shows how each content type is to be used in the various tea
 
 ### Lectures
 
-### Practical sessions
+### Practical Sessions
 
 ### Exams
 
-### Assignments
+### Projects
```

Comment:
The super linter complaints about this, that's why I renamed it differently. Universities typically call them "assignments"; "projects" is used for something more broader which typically involves working teams or doing some sort of research before doing something applicable. We can store this type of activity as a "Project" but call it an "Assignment" at the teaching stage.

Reply:
I see. I'll rename it.